public/locales/en/modules/media-server.json

```diff
@@ -6,6 +6,7 @@
       "title": "Settings for media server widget"
     }
   },
+  "loading": "Homarr is loading streams...",
```

Comment:
Just loading steams makes more sense. No need for "…" (anywhere)

Reply:
I agree that we can move the dots to the actual string. The App name can be replaced with a variable